core-bundle/src/Resources/contao/dca/tl_form_field.php

```diff
@@ -731,7 +732,7 @@ public function getFields()
 
 		foreach ($GLOBALS['TL_FFL'] as $k=>$v)
 		{
-			if ($this->User->hasAccess($k, 'fields'))
+			if (System::getContainer()->get('security.helper')->isGranted(ContaoCorePermissions::USER_CAN_ACCESS_FIELD_TYPE, $k))
```

Comment:
Retrieving the security helper should be done outside the loop.

Reply:
fixed in 45800a6d689de1e18415bcdb67a1a62af3fa8d0f